python/utils/train_model.py

```diff
@@ -184,82 +220,111 @@ def flat_accuracy(preds, labels):
 valid_sampler = SequentialSampler(valid_data)
 valid_dataloader = DataLoader(valid_data, sampler=valid_sampler, batch_size=BATCH_SIZE)
 
-print('Loaded training and validation data into DataLoaders successfully.')
+print("Loaded training and validation data into DataLoaders successfully.")
 
 ###
 
 # Initialize model
-model = BertForTokenClassification.from_pretrained('bert-base-cased',
-                                                   num_labels=NUM_LABELS)
+model = BertForTokenClassification.from_pretrained(
+    "bert-base-cased", num_labels=NUM_LABELS
+)
 
 model.to(device)
 
-print(f'Initialized model and moved it to {device}.')
+print(f"Initialized model and moved it to {device}.")
 
 ###
 
 # Set hyperparameters (optimizer, weight decay, learning rate)
 
 if FULL_FINETUNING:
     param_optimizer = list(model.named_parameters())
-    no_decay = ['bias', 'gamma', 'beta']
+    no_decay = ["bias", "gamma", "beta"]
     optimizer_grouped_parameters = [
-        {'params': [p for n, p in param_optimizer
-                    if not any(nd in n for nd in no_decay)],
-         'weight_decay_rate': 0.01},
-        {'params': [p for n, p in param_optimizer
-                    if any(nd in n for nd in no_decay)],
-         'weight_decay_rate': 0.0}
+        {
+            "params": [
+                p for n, p in param_optimizer if not any(nd in n for nd in no_decay)
+            ],
+            "weight_decay_rate": 0.01,
+        },
+        {
+            "params": [
+                p for n, p in param_optimizer if any(nd in n for nd in no_decay)
+            ],
+            "weight_decay_rate": 0.0,
+        },
     ]
 else:
     param_optimizer = list(model.classifier.named_parameters())
     optimizer_grouped_parameters = [{"params": [p for n, p in param_optimizer]}]
 
 optimizer = Adam(optimizer_grouped_parameters, lr=3e-5)
 
-print('Initialized optimizer and set hyperparameters.')
+print("Initialized optimizer and set hyperparameters.")
 
 ###
 
+epoch = 0
 for _ in trange(EPOCHS, desc="Epoch"):
-
+    epoch = epoch + 1
```

Comment:
I know I wrote this but.. should be "epoch += 1"

Reply:
forgot to include this comment in the review: the train_model should be imported as a module, and the code below the functions that runs it should be in a separate main.py file

Reply:
Nina from meeting: "not urgent, just one of those things; as I'm reviewing I have to comment, but I figured that was the case"